solr/core/src/test/org/apache/solr/response/TestPrometheusResponseWriter.java

```diff
@@ -34,57 +38,104 @@
 import org.junit.ClassRule;
 import org.junit.Test;
 
-/** Tests the {@link PrometheusResponseWriter} behavior */
-@LuceneTestCase.AwaitsFix(bugUrl = "https://issues.apache.org/jira/browse/SOLR-17368")
 public class TestPrometheusResponseWriter extends SolrTestCaseJ4 {
-
   @ClassRule public static SolrJettyTestRule solrClientTestRule = new SolrJettyTestRule();
   public static JettySolrRunner jetty;
```

Comment:
I think this field serves no purpose; we can get a jetty if needed from the rule on the fly.  Static fields in tests can have a maintenance burden to ensure they get null'ed.